File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    
    for command in commands:
        c1 = command[0]-1
        c2 = command[1]
        c3 = command[2]-1
        
        c = sorted(array[c1:c2])
        c = c[c3]
        
        answer.append(c)
    
    return answer

File: 프로그래머스/1/42748. K번째수/K번째수.java
import java.util.Arrays;

class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        
        for(int i=0;i<commands.length;i++){
            int c1 = commands[i][0];
            int c2 = commands[i][1];
            int c3 = commands[i][2];
            
            int[] temp = Arrays.copyOfRange(array, c1-1, c2);
            Arrays.sort(temp);
            answer[i] = temp[c3-1];
        }
        
        return answer;
    }
}